docs/pages/guides/schema-validation.mdx

```diff
@@ -0,0 +1,124 @@
+
+## How Schema Validation works
+As described above, the main purpose of Schema Validation is to prevent corrupted data from being loaded from the server. To accomplish this, two things need to happen. First, we need to ensure that data loaded into a Room matches the underlying Storage Tree and defined schema (if it exists). Second, we will introduce the rule that each operation or "Op" that occurs in the room which changes the Storage tree, will either be accepted or rejected.  
+
+When a Room is first loaded, the key `liveblocksSchema` from Storage is inspected: if no value is present, then schema validation is not enforced.  If there is a value, then that value of the schema is parsed and assigned to the Room where its values are distributed to the node tree.
+
+<!-- Insert an animation here !-->
+
+If schema validation is not enforced, all operations in the room are accepted. 
+For example, imagine that a client performs an operation that updates the Storage Tree:  regardless of whether or not this could break the application, Liveblocks has no opinion on whether or not this operation should be completed and accepts the incoming operation.  After the operation is completed, the updated Storage Tree is forwarded to other users in the room. 
+
+In the case where schema validation is enforced, when the client performs an operation that updates the Storage Tree, it is sent to the Liveblocks server which verifies that the updated Storage Tree is consistent with the defined schema attached to the Room.  If the verification succeeds, the Op is acknowledged by the initial client, and the Storage Tree is updated and forwarded to other users (clients) in the room.  If the operation verification fails, a `RejectedOpsServerMsg` is sent to the client.  The Storage Tree is not updated. In this instance, it is the responsibility of the developer to resolve the error.
+
+## Create a schema
+To create a schema, you will need to POST to the schemas endpoint:
+```ts
+POST https://api.liveblocks.io/v2/schemas
+{
+  "id": "some-id",
+  "name": "Your schema name",
+  "data": `{
+    // your data schema will go here
+  }`,
+}
+
+type Body = {
+  id: string, // MAX_LENGTH = 64 / allowed characters az-
+  name: string, // MAX_LENGTH = 256
+  data: string,
+}
+```
+Note that the data type will be of type string because the schema will be string interpolated. 
+The operation above creates the schema, but it has not yet been assigned to a Room.  To attach this schema to a Room, you will POST to the rooms endpoint:
+```ts
+POST https://api.liveblocks.io/v2/rooms/{id}/schema
+{
+  "schema": "some-id"
+}
+```
+## The validation process in practice
+If you would like to work through an example, you can follow along by using the Multiplayer Form example.
```

Comment:
```suggestion
If you would like to work through an example, you can follow along by using the [Multiplayer Form](/examples/multiplayer-form/nextjs) example.
```

So that people know where to go if they're curious about it

Reply:
Side note, @stevenfabre: can you confirm that this example should be renamed `Collaborative form` and not `Multiplayer form` after our repositioning?

Reply:
I can check in the preview to validate, else, I can rename as a separate PR @stevenfabre and update convention here

Reply:
In preview I still see multiple examples being referred to as Mutliplayer (including the form)